windows/benji.py

```diff
@@ -417,6 +418,65 @@ def main():
 			speak.say("Sorry Graph can not be Plotted")
 			speak.runAndWait()
 
+	#Box Office Status
+	elif link[-1] == "boxoffice":
+		try:
+			url="https://in.bookmyshow.com/"+ link[0] +"/movies/nowshowing"

```

Comment:
Is there any rate limiting on the `bookmyshow.com`?

Reply:
We should add a disclaimer that Benji assumes the location to be in `India`.
Book my show works in `Indonesia | UAE | Sri Lanka` as well.

Reply:
I don't think that there is any rate limiting on the `bookmyshow.com`.
I used only simple web scrapping. And we don't have to login into `bookmyshow.com` Account.
I'll write that this feature is only available within `Indian Region`.